src/arch/arm/rte_pause.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2017 Cavium, Inc
+ */
+
+#ifndef _RTE_PAUSE_ARM_H_
+#define _RTE_PAUSE_ARM_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "arch/arm/rte_pause_64.h"
```

Comment:
what is the need of having rte_pause.h and rte_pause_64.h if functionality wise both are same. We can have one heqder file only here?

Reply:
If there is no need to support arm32, we can combine two files into one. Actually  there also include rte_pause_32.h， but I remove it.

Reply:
Better not to combine two files. Keep both the file separate.